### ISSUE TITLE
Signature and image upload do not populate pdf template correctly

### ISSUE THREAD

**Issue by mrherman** [jhpyle/docassemble#602]
When I have a pdf template with a signature and image field, the resulting pdf does not put the signature variable and image variable in the right template form field, instead it seems to put both in to the last form field.  In the below short example, the signature and the front of the insurance card fill the same signature form block that I want the front of the insurance card to go.

```
metadata:
  title: Daycamp Medical Form 2023
  short title: Medical Form
---
mandatory: True
question: |
  Your Medical Form is Complete
subquestion: |
  Thank you
buttons:
  - Exit: exit
  - Restart: restart
attachments:
  - name: Medical Form
    filename: Medical_Form
    pdf template file: 680-001_AB.pdf
    fields: 
      "Full name": ${ full_name }
      "Signature1": ${ signature }
      "CardFront": ${ card_front }
---      
question: Scout Info
subquestion: Tell us about the participant
fields:
  - Name: full_name
---
question: Please Sign
signature: signature
---
question: Insurance Card - Front
fields:
  - Picture: card_front
    datatype: environment 
```

**Comment by jhpyle** [comment]
Sorry about that. I think I have fixed it in 1.4.24.

**Comment by mrherman** [comment]
Tested it, works as expected.
